packages/offline-packages.yaml.tmpl

```diff
@@ -52,8 +52,83 @@ editions:
       tags:
         - "{{ .Release.version }}-community"
     routers:
-      - description: "Started from v7.6.0 to latest added tiproxy compare with v7.5.0"
-        if: {{ semver.CheckConstraint ">=7.6.0-0" .Release.version }}
+      - description: "Started from v8.4.0"
```

Comment:
Why here add that much contents also why changed it from v7.6.0 to v8.4.0-0?

Reply:
@Benjamin2037 
The diff page is not pretty, you can ignore it: it's a new route.

<img width="542" alt="image" src="https://github.com/user-attachments/assets/ed7966be-0a6b-4c96-803b-cecde71357b5">